sdk/attestation/Azure.Security.Attestation/tests/Samples/AttestationServiceAttestationSamples.cs

```diff
@@ -177,12 +181,14 @@ public async Task SettingAttestationPolicy()
             // When the attestation instance is in Isolated mode, the ResetPolicy API requires using a signing key/certificate to authorize the user.
```

Comment:
Should we consider an alternative to `CollectionAssert`  in the snippet?  It will potentially be unfamiliar to developers not using NUnit and isn't likely something that you'd see in a production application outside of it's test suite.

Reply:
Using SequenceEqual from Linq and Debug.Assert instead.